### PR TITLE
Create sudeepjhs.json

### PR DESCRIPTION
**Pull request by sudeepjhs**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/sudeepjhs.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Sudeep Agarwal",
+  "bio" : "Innovative Software Developer | JavaScript, ReactJS, Node.js, NextJS Enthusiast | Full Stack Developer | Outsystems Learner"
```
Comment (sital002) [inline data/sudeepjhs.json:3]:
```suggestion
  "bio" : "Innovative Software Developer | JavaScript, ReactJS, Node.js, NextJS Enthusiast | Full Stack Developer | Outsystems Learner",
```
There is a comma missing

File: data/sudeepjhs.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "links":[
+    {
+      "group": "Socials Primary",
+      "name": "LinkedIn: Let's Connect",
+      "url": "https://www.linkedin.com/in/sudeepagarwaljhs/",
+      "icon": "FaLinkedin",
+      "color": "#00ACEE"
+    },
```
Comment (sital002) [inline data/sudeepjhs.json:11]:
```suggestion
    }
```
The trailing comma isnot required

File: data/sudeepjhs.json (hunk 1 above)
Comment (sudeepjhs) [reply data/sudeepjhs.json:3]: ok noted

File: data/sudeepjhs.json (hunk 2 above)
Comment (sudeepjhs) [reply data/sudeepjhs.json]: Ok, thanks

**Pull request closed without merging**